### PR TITLE
fixed injury broken ankle bug

### PR DESCRIPTION
**Pull request by RikerW**

### CODE REVIEW

File: survive.rb [hunk 1]
```diff
@@ -145,9 +145,10 @@ def worm!(range=25, new=true)
   
   case tiles[size[0] / 2][size[1] / 2]
     when 'S'
-      if $water > 50 && rand([75 - $water, 1].max).zero?
+      if $water > 20000000 && rand([75 - $water, 1].max).zero?
```
Comment (newbiedoodle) [inline survive.rb]: The thing that concerns me about this is that changing the check to be so high... basically invalidates it. The whole point is that it only happens if you're carrying a lot of water.

Comment (RikerW) [inline survive.rb]: oops.  i did that while testing, sorry.

**Comment by RikerW** [comment]
I don't ruby well, feel free to change this or reject.

**Comment by quartata** [comment]
By the way, I think that this should be lower than 50. An average human needs a gallon of water a day in the desert, so if we assume that one unit of water is one gallon, the person should really only be able to carry 15 units of water (about 120 pounds) before bad things happen.

**Comment by quartata** [comment]
Ideally there should be a bag of holding or similar rare magic item to let them carry more too.

**Comment by newbiedoodle** [comment]
@quartata The numbers are all subject to change, especially since I'll soon(ish) be changing how long one turn is. I might also increase water consumption. As it is, I'm accepting the pull request because it fixes the bug.

**Comment by RikerW** [comment]
@quartata  nethack much? :P

**Pull request merged**